executor/binary.go

```diff
@@ -115,7 +115,9 @@ func (b *Binary) Run() error {
 		}
 	}
 
+	binarySpec := b.Spec
 	lgr.Debug.Println("start executing binary command.")
+	lgr.Debug.Println("Namespace:", binarySpec.Namespace, ",Name:", binarySpec.Name, ",version:", binarySpec.Version)
```

Comment:
In accordance with the others.
```suggestion
	lgr.Debug.Println("Namespace:", binarySpec.Namespace, ",Name:", binarySpec.Name, ",Version:", binarySpec.Version)
```